apisix/core/request.lua

```diff
@@ -334,6 +335,21 @@ function _M.get_body(max_size, ctx)
 end
 
 
+function _M.get_request_body_table()
```

Comment:
Decode body according content-type or rename the method, e.g. get_json_request_body_table?